### ISSUE TITLE
Robolectric support

### ISSUE THREAD

**Issue by djordjeh** [kotest/kotest#3505]
I just tried to update Kotest from v4.5.0 -> 5.6.1 and I've noticed that all my tests with annotation Robolectric are failing. I see kotest team has archived robolectric extension and there is no any documentation available on the website. 

So, what the final verdic on robolectric and kotest? Is there any way to use them together with the latest version?

Thanks for any help.

**Comment by LeoColman** [comment]
We've decided to not add official support to Android for the time being, as we don't have enough Android-Skilled people in Kotest's team.

That includes Robolectric. We were never able to get it to a stable and working 1.0.0 version, and maintaining it was incompatible with our knowledge and goals as Kotest, so that was parked.

----

Given that, I'm still an Android + Kotest lover, and started trying to maintain Kotest's archived Android features: https://github.com/LeoColman/kotest-android . I've created https://github.com/LeoColman/kotest-android/issues/2 to eventually implement and maintain a Robolectric Extension.

----

> So, what the final verdic on robolectric and kotest?

Kotest Team is no longer officially supporting Robolectric, Android Runner, extensions or assertions.


> Is there any way to use them together with the latest version?
- Writing/Copying and adapting the old extension to your current code
- Use an extension from a 3rd party that is still up-to-date (none that I know at the moment)
